apps/labs/posts/pydata-theme-a11y-fixes.mdx

```diff
@@ -0,0 +1,479 @@
+| ![Before: bottom side of focus ring hidden by currently selected version in version dropdown](/posts/pydata-theme-a11y-fixes/focus-ring-version-dropdown-before.png) | ![Before: same thing with the additional nav bar links dropdown: focus ring partially eclipsed](/posts/pydata-theme-a11y-fixes/focus-ring-more-dropdown-before.png) | ![Before: right side of focus ring hidden by vertical divider in left sidebar table of contents](/posts/pydata-theme-a11y-fixes/focus-ring-left-toc-before.png) | ![Before: the previous theme focus ring sometimes overlapped Bootstrap's focus ring, creating an odd halo effect](/posts/pydata-theme-a11y-fixes/focus-ring-header-nav-link-before.png) |
+| ![After: all four sides of focus ring fully visible](/posts/pydata-theme-a11y-fixes/focus-ring-version-dropdown-after.png)                                           | ![After: focus ring fully visible, all four sides](/posts/pydata-theme-a11y-fixes/focus-ring-more-dropdown-after.png)                                               | ![After: right side plus other three sides all visible in left sidebar table of contents](/posts/pydata-theme-a11y-fixes/focus-ring-left-toc-after.png)         | ![After: now its just the theme's focus ring, a thick, purple, rounded focus ring](/posts/pydata-theme-a11y-fixes/focus-ring-header-nav-link-after.png)                                 |
+
+There isn't a single GitHub pull request that I can point to in which all of
+these focus ring issues were fixed. They were done over many weeks and PRs,
+including but not limited to:
+[#1415](https://github.com/pydata/pydata-sphinx-theme/pull/1415),
+[#1555](https://github.com/pydata/pydata-sphinx-theme/pull/1555),
+[#1561](https://github.com/pydata/pydata-sphinx-theme/pull/1561),
+[#1564](https://github.com/pydata/pydata-sphinx-theme/pull/1564),
+[#1582](https://github.com/pydata/pydata-sphinx-theme/pull/1582),
+[#1589](https://github.com/pydata/pydata-sphinx-theme/pull/1589),
+[#1778](https://github.com/pydata/pydata-sphinx-theme/pull/1778).
+
+## And more!
+
+This blog post is not meant to capture all of the accessibility work that we did
+over the past six months, but rather my hope is that I have given you a taste of
+what this work looks like from the developer side of things.
+
+If you would like to take a more exhaustive look, your best option would be to
+look at all of the [PST issues and PRs tagged with
+accessibility](https://github.com/pydata/pydata-sphinx-theme/issues?q=label%3A%22tag%3A+accessibility%22).
+If you find anything you would like to see written about in a blog post like
+this one, please let us know!
+
+## Accessibility in this Blog Post
```

Comment:
This is a cool section. Even though it felt separate from the rest of the blog post to me, I would keep it. For example, I totally went to check the description for the code and was pleasantly surprised to find Codepens instead of images.